source/features/branch-buttons.tsx

```diff
@@ -40,7 +40,7 @@ async function getTagLink(): Promise<'' | HTMLAnchorElement> {
 
 	const link = <a className="btn btn-sm btn-outline tooltipped tooltipped-ne">{icons.tag()}</a> as unknown as HTMLAnchorElement;
 
-	const currentBranch = select('.branch-select-menu .css-truncate-target')!.textContent;
```

Comment:
Actually I'm curious about why we chose to select the current branch like this before. AFAIK, it would only selected the truncated branch name but not the full name. Is there any concerns I don't know about?

Reply:
The name is only truncated via CSS so in the DOM it's still full